<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageLogin">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="2" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>59</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="m_pWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>528</width>
           <height>380</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>528</width>
           <height>407</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget{border-radius:12px; background-color: #ffffff}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>16</number>
          </property>
          <property name="leftMargin">
           <number>59</number>
          </property>
          <property name="topMargin">
           <number>24</number>
          </property>
          <property name="rightMargin">
           <number>59</number>
          </property>
          <property name="bottomMargin">
           <number>24</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>20</number>
            </property>
            <item>
             <widget class="QWidget" name="m_pWidgetIcon" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>410</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>410</width>
                <height>32</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget{ background-color: #ffffff; }</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <property name="spacing">
                <number>16</number>
               </property>
               <property name="leftMargin">
                <number>56</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>56</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="m_pIcon">
                 <property name="styleSheet">
                  <string notr="true">border-left-width:0px; border-right-width:0px; </string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="m_pWidgetIp" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>410</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>410</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget{border:1px solid #d2d2d2; outline:5px; border-radius:4px; background-color: #ffffff; }</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <property name="spacing">
                <number>16</number>
               </property>
               <property name="leftMargin">
                <number>16</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="m_pIconIp">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-left-width:0px; border-right-width:0px; border-radius:0px; </string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="m_pIp">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-left-width:0px; font:13px;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="m_pWidgetUser" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>410</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>410</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget{border:1px solid #d2d2d2; outline:5px; border-radius:4px; background-color: #ffffff; }</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <property name="spacing">
              <number>16</number>
             </property>
             <property name="leftMargin">
              <number>16</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="m_pIconUser">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-left-width:0px; border-right-width:0px; border-radius:0px; </string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="m_pUser">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">border-left-width:0px; font:13px;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="m_pWidgetPasswd" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>410</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>410</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget{border:1px solid #d2d2d2; outline:5px; border-radius:4px; background-color: #ffffff; }</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="spacing">
              <number>16</number>
             </property>
             <property name="leftMargin">
              <number>16</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="m_pIconPasswd">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-left-width:0px; border-right-width:0px; border-radius:0px; </string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="m_pPasswd">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">border-left-width:0px; font:13px;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>20</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="m_pRememberPW">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">font:12px</string>
                </property>
                <property name="text">
                 <string>记住密码</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="m_pTip">
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true"> color:red;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="m_pLogin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>410</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>410</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>59</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>208</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>129</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>209</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="m_pCopyR">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>528</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>528</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12px &quot;Noto Sans CJK SC&quot;; color: rgb(145, 145, 145)</string>
         </property>
         <property name="text">
          <string>Copyright © 2025 湖南麒麟信安 版权所有</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageFrame">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
